.github/workflows/scan.yml

```diff
@@ -0,0 +1,48 @@
+# Scan the code in this repository; publish results to
+# https://github.com/intel/openvino-rs/security/code-scanning.
+
+name: Code Scan
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+  schedule:
+    - cron: "3 2 * * 1"
```

Comment:
Why does this need to run as a cron if the analysis tool is static? Why not just run once when a new patchset is initiated or updated?

Reply:
Might as well run it once a week in case the underlying tools change and the repository has not had a PR in a while.